avm/res/container-service/managed-cluster/main.bicep

```
@@ -599,6 +599,7 @@ resource managedCluster 'Microsoft.ContainerService/managedClusters@2024-03-02-p
       osDiskSizeGB: profile.?osDiskSizeGB
       osDiskType: profile.?osDiskType
       osType: profile.?osType ?? 'Linux'
+      osSKU: profile.?osSku ?? 'AzureLinux'
```

Let's bring some consistency here and let us always use `osSKU`.
default value is Ubuntu as per documentation:
> Specifies the OS SKU used by the agent pool. If not specified, the default is Ubuntu if OSType=Linux or Windows2019 if OSType=Windows. And the default Windows OSSKU will be changed to Windows2022 after Windows2019 is deprecated.

```suggestion
      osSKU: profile.?osSKU'
```

---

Also please update osSku -> osSKU in the type definition, parameter definition and in agentPool.bicep
alternatively, I already have the changes in a branch of mine.
https://github.com/JPEasier/bicep-registry-modules/tree/users/jpeasier/avm-aks

---

If you have the changes in your Branch already then, we can use that, and I can close this PR. When would you be pushing your branch?